moveit_planners/chomp/chomp_motion_planner/src/chomp_optimizer.cpp

```diff
@@ -953,12 +960,12 @@ void ChompOptimizer::performForwardKinematics()
           {
             state_is_in_collision_[i] = true;
             // if(is_collision_free_ == true) {
-            //   RCLCPP_INFO(LOGGER,"We know it's not collision free " << g);
```

Comment:
Maybe remove if it commented out anyways

Reply:
:+1: